tensorflow/core/kernels/matmul_op_test.cc

```diff
@@ -80,6 +84,28 @@ class FusedMatMulOpTest : public OpsTestBase {
           return device.device_type() == DEVICE_GPU;
         });
 
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM //This needed for CPU only builds
```

Comment:
There's no way to check GPU availability at runtime?

Reply:
I had to do this because the CPU only build would fail